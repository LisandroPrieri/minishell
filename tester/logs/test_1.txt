========== Test 1 ==========
Command: 
======================================
Output: cat: infile1: No such file or directory
cat: infile2: No such file or directory
cat: infile3: No such file or directory
======================================
Return Value: 
========== End of Test 1 ==========
No expected outfile found: ./expected_outputs/test_1.txt
