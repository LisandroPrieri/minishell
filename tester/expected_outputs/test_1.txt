========== Test 1 ==========
Command: cat infiles/infile1 infiles/infile2 infiles/infile3
======================================
Output: abcdefghi
======================================
Return Value: 0
========== End of Test 1 ==========
